front-end/src/Routes/AdminRoute.js

```diff
@@ -1,16 +1,17 @@
-import React from 'react'
-import NumberContainer from '../Numbers/NumberContainer'
-import NumberListFetcher from '../Numbers/NumberListFetcher'
-import AdminView from '../AdminBar/AdminView'
-import { AppContainer } from '../utils/styles'
+import React from 'react';
+import NumberContainer from '../Numbers/NumberContainer';
+import NumberListFetcher from '../Numbers/NumberListFetcher';
+import AdminView from '../AdminBar/AdminView';
+import { AppContainer } from '../utils/styles';
 
-const AdminRoute = () => (
+const NotifyRoute = () => (
```

Comment:
Shouldn't this be called `AdminRoute`? :thinking:

Reply:
Whooops! thanks!